conformance/conformance.proto

```diff
@@ -78,7 +78,9 @@ message ConformanceRequest {
   // Which format should the testee serialize its message to?
   WireFormat requested_output_format = 3;
 
-  // should be set to either "proto2" or "proto3"
+  // The full name for the test message to use; either
```

Comment:
Maybe better to say: "for the moment, either:"

We might have reason to add others in the future. Implementations that can lookup messages by name should just do that.